packages/openapi-generator/src/parser/operation.ts

```diff
@@ -72,13 +72,33 @@ function isPlaceholder(pathPart: string): boolean {
   return /^\{.+\}$/.test(pathPart);
 }
 
+function validatePlaceholder(pathPart: string): boolean {
+  return /^\{[^{}]+\}$/.test(pathPart);
+}
+
+function validatePathPattern(
+  pathPattern: string,
+  placeholders: string[]
+): boolean {
+  return (
+    pathPattern.includes('?') ||
+    placeholders.some(placeholder => !validatePlaceholder(placeholder))
```

Comment:
Is place holder a two word thing so `placeHolder` I am not sure, perhaps you can find out.

Reply:
[placeholder](https://dictionary.cambridge.org/dictionary/english/placeholder) is a word.